cluster/images/etcd-empty-dir-cleanup/etcd-empty-dir-cleanup.sh

```diff
@@ -17,7 +17,7 @@
 echo "Removing empty directories from etcd..."
 
 cleanup_empty_dirs () {
-  if [[ $(${ETCDCTL} ls $1) ]]; then
+  if [[ "$(${ETCDCTL} ls $1)" ]]; then
```

Comment:
It's not exactly a built-in in busybox:
```
$ docker run --rm -i -t busybox which [[
/bin/[[
```

That's probably why this command broke when switched from bash to busybox's ash.